src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorManager.class.php

```diff
@@ -225,42 +225,16 @@ public function HandleExternalFunctionCall($sFunctionName, $aParameters = array(
     }
 
     /**
-     * load the table editor class that is configured in cms_tbl_conf.
-     *
-     * @deprecated the hardcoded switch by table name is deprecated and can be deleted after upgrading all customers
-     *
-     * @return TCMSTableEditor
+     * load the table editor class that is configured in cms_tbl_conf.*
      */
-    public function &TableEditorFactory()
+    public function &TableEditorFactory(): TCMSTableEditor
```

Comment:
We can not the the return type here, because of a possible BC break with extended code.

Reply:
ok